Modules/clinic/posixmodule.c.h

```diff
@@ -3853,6 +3853,77 @@ os_write(PyObject *module, PyObject *const *args, Py_ssize_t nargs)
     return return_value;
 }
 
+#if defined(__APPLE__)
+
+PyDoc_STRVAR(os__fcopyfile__doc__,
+"_fcopyfile($module, infd, outfd, /)\n"
+"--\n"
+"\n"
+"Efficiently copy the content of 2 file descriptors (OSX only).");
+
+#define OS__FCOPYFILE_METHODDEF    \
+    {"_fcopyfile", (PyCFunction)os__fcopyfile, METH_FASTCALL, os__fcopyfile__doc__},
+
+static PyObject *
+os__fcopyfile_impl(PyObject *module, int infd, int outfd);
+
+static PyObject *
+os__fcopyfile(PyObject *module, PyObject *const *args, Py_ssize_t nargs)
+{
+    PyObject *return_value = NULL;
+    int infd;
+    int outfd;
+
+    if (!_PyArg_ParseStack(args, nargs, "ii:_fcopyfile",
+        &infd, &outfd)) {
+        goto exit;
+    }
+    return_value = os__fcopyfile_impl(module, infd, outfd);
+
+exit:
+    return return_value;
+}
+
+#endif /* defined(__APPLE__) */
+
+#if (defined MS_WINDOWS)
```

Comment:
Misplaced parenthesis? Even if this happens to work (oh the wonders of C!), I can't see a reason to spell it this way.